Services/DataBaseService/Commands/DeleteUserCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DataBaseService.Interfaces;
+using DTO;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace DataBaseService.Commands
+{
+    public class DeleteUserCommand : ICommand<Guid>
```

Comment:
Isn't this make conflicts with create logic? 
Now we return id in order to check that user was created . But then they will be void both